.travis.yml

```diff
@@ -95,11 +91,11 @@ matrix:
         # time.
 
         - os: linux
-          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.12
+          env: PYTHON_VERSION=3.6 NUMPY_VERSION=1.12
```

Comment:
what about python 3.7? I suppose that's OK to do it in a separate PR.

Reply:
You think there's value in adding Python 3.7 builds? Or should some be replaced?

In my experience passing on Python 3.6 and failing on Python 3.7 is super rare, not worth the 🔌 , better to save a few 🌴 🎋 🎄 🌳 🌲 s. But I don't care much, just le me know what you suggest or put it directly.

Reply:
I suggest to replace. In other packages we scatter versions around to cover the combinations. E.g. no need to test different numpy versions with the same python, change both to you can cover more (and can rely on the assumption that all upstream packages are testing against the remaining combinations, that is a sensible assumption).

Reply:
Also I would consider dropping a few numpy versions from support, 1.10 is 3 years old already. Or if you don't want it then test the oldest one you support, but then cover the newer ones rather than test all the old ones.